common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -200,6 +207,25 @@ def get_html(self):
                 val_profiles = ["youtube", "desktop_webm", "desktop_mp4"]
                 val_video_urls = edxval_api.get_urls_for_profiles(self.edx_video_id, val_profiles)
 
+                # download youtube transcript for html5 video
+                if not self.sub and val_video_urls["youtube"]:
+                    #cache the transcript request
+                    cache_key = 'html5_' + val_video_urls["youtube"]
+                    html5_transcript_cache = cache.get(cache_key)
```

Comment:
What happens if there isn't sub, and we cache this response, but then a course team adds a sub? Will we keep hitting the cache and saying there isn't a sub?